pages/index.vue

```diff
@@ -1,28 +1,99 @@
 <template>
-  <v-container
-    fluid
-    fill-height>
-    <v-layout
-      row
-      align-center
-      justify-center>
-      <div class="background-div">
-        <img
-          src="../assets/img/calendarLogo.png"
-          class="background-image">
-      </div>
-    </v-layout>
-  </v-container>
+  <v-card>
```

Comment:
Ist die v-card auserhalb notwendig?
Wenn nicht würde ich die entfernen weil sich sonst die Kacheln nicht richtig in den Hintergrund einfügen?